packages/amqp/package.json

```diff
@@ -25,7 +25,7 @@
     "prepublishOnly": "npm run build:release"
   },
   "dependencies": {
-    "@lokalise/node-core": "^13.1.0",
+    "@lokalise/node-core": "^13.3.0",
```

Comment:
🟡 Should we also increase mqt core version from `>=15` to `>=18`?

Reply:
(Same question fro SNS and SQS)

Reply:
You mean from compatibility perspective? `CommonLogger` is compatible with old interface so older version should work fine

Reply:
Is it? I am just following the migration guide so please let me know if I missing something:
> If you are using `pino` logger, or any other logger compatible with `CommonLogger` interface, you don't need to do anything. Otherwise, there are serveral properties that need to be provided compared to the old interface:

So I guess it depends on the logger the user is using

Reply:
That's correct, but it's the case if you want to upgrade to the new version.
So, if you are upgrading, your logger needs to fulfil specific requirements. But if it's already meeting those requirements, it can also be used with old version of the library.

So on this example: if you upgrade `amqp`, but keep old `core`, you will need to use new logger for `amqp` and old `core` will accept it

Reply:
Oh I see, sorry for the confusion 😅 all good, thanks for the explanation 🙏